### PR TITLE
Document JSON-RPC recording

### PR DESCRIPTION
**Pull request by ann0see**
**Short description of changes**

Documents the possibility the JSON-RPC changed the recording state in the GUI.
CHANGELOG: SKIP 

**Context: Fixes an issue?**

Fixes: #3037
**Does this change need documentation? What needs to be documented and how?**

No.

**Status of this Pull Request**
Ready for review. I believe it's not much. 

**What is missing until this pull request can be merged?**
In fact depends on: #2998 

## Checklist

-  [x] I've verified that this Pull Request follows the [general code principles](https://github.com/jamulussoftware/jamulus/blob/main/CONTRIBUTING.md#jamulus-projectsource-code-general-principles)
-  [x] I tested my code and it does what I want
-  [x] My code follows the [style guide](https://github.com/jamulussoftware/jamulus/blob/main/CONTRIBUTING.md#source-code-consistency) 
-  [x] I waited some time after this Pull Request was opened and all GitHub checks completed without errors. 
-  [x] I've filled all the content above

### CODE REVIEW

File: src/serverdlg.cpp [hunk 1]
```diff
@@ -136,7 +136,7 @@ CServerDlg::CServerDlg ( CServer* pNServP, CServerSettings* pNSetP, const bool b
 #ifdef _WIN32
         + tr ( "Recording has been switched off by the UI checkbox." )
 #else
-        + tr ( "Recording has been switched off, either by the UI checkbox or SIGUSR2 being received." )
+        + tr ( "Recording has been switched off, by UI checkbox, SIGUSR2 or JSON-RPC." )
```
Comment (pljones) [inline src/serverdlg.cpp:139]: JSON-RPC is available on Windows, too.  It's only SIGUSR2 that doesn't apply.

Comment (ann0see) [reply src/serverdlg.cpp]: Thanks. Fixed.

**Review by pljones — APPROVED** [review]

**Comment by ann0see** [comment]
Due to the review and the small nature of this PR: Merging.

**Comment by ann0see** [comment]
Actually, it does not depend on the weblate PR.

**Pull request merged**